source/bin.ts

```diff
@@ -1,16 +1,20 @@
 #!/usr/bin/env node
 
-import validate from './index.js'
+import validate, { IsValidDirectory } from './index.js'
 const path = process.argv[2] || process.cwd()
 
 validate(path)
-	.then((path) => {
-		console.log(`${path} is valid`)
+	.then((valid: IsValidDirectory) => {
+		if (valid[0]) {
+			console.log(`${path} is valid`)
+		} else {
+			console.error(
```

Comment:
interpolating of interpolating the paths, pass it as an additional argument to `console.error` that way it will get a nice display and syntax highlighting

`following paths:\n${valid[1]}` -> `following paths:', invalidPaths`